src/ray/object_manager/pull_manager.cc

```diff
@@ -48,6 +48,7 @@ PullManager::PullManager(
 uint64_t PullManager::Pull(const std::vector<rpc::ObjectReference> &object_ref_bundle,
                            BundlePriority prio,
                            const std::string &task_name,
+                           bool is_retry,
```

Comment:
Nit: Could we add a new typedef like TaskMetricsKey? It is not very clear what `is_retry` means in the `PullManager` context, could easily be confused with retrying a pull.